mlir_graphblas/src/test/GraphBLAS/structuralize_apply.mlir

```diff
@@ -361,31 +361,43 @@ module {
-// CHECK:           %[[VAL_1:.*]] = constant 1 : i8
-// CHECK:           %[[VAL_2:.*]] = graphblas.apply_generic %[[VAL_0]] : tensor<?x?xi8, #sparse_tensor.encoding<{ dimLevelType = [ "dense", "compressed" ], dimOrdering = affine_map<(d0, d1) -> (d0, d1)>, pointerBitWidth = 64, indexBitWidth = 64 }>> to tensor<?x?xi8, #sparse_tensor.encoding<{ dimLevelType = [ "dense", "compressed" ], dimOrdering = affine_map<(d0, d1) -> (d0, d1)>, pointerBitWidth = 64, indexBitWidth = 64 }>>  {
-// CHECK:           ^bb0(%[[VAL_3:.*]]: i8):
-// CHECK:             %[[VAL_4:.*]] = divi_signed %[[VAL_1]], %[[VAL_3]] : i8
-// CHECK:             graphblas.yield transform_out %[[VAL_4]] : i8
+// CHECK:           %[[VAL_1:.*]] = constant 7 : i8
+// CHECK:           %[[VAL_2:.*]] = constant 1 : i8
+// CHECK:           %[[VAL_3:.*]] = graphblas.apply_generic %[[VAL_0]] : tensor<?x?xi8, #sparse_tensor.encoding<{ dimLevelType = [ "dense", "compressed" ], dimOrdering = affine_map<(d0, d1) -> (d0, d1)>, pointerBitWidth = 64, indexBitWidth = 64 }>> to tensor<?x?xi8, #sparse_tensor.encoding<{ dimLevelType = [ "dense", "compressed" ], dimOrdering = affine_map<(d0, d1) -> (d0, d1)>, pointerBitWidth = 64, indexBitWidth = 64 }>>  {
+// CHECK:           ^bb0(%[[VAL_4:.*]]: i8):
+// CHECK:             %[[VAL_5:.*]] = shift_right_signed %[[VAL_4]], %[[VAL_1]] : i8
+// CHECK:             %[[VAL_6:.*]] = addi %[[VAL_5]], %[[VAL_4]] : i8
+// CHECK:             %[[VAL_7:.*]] = xor %[[VAL_5]], %[[VAL_6]] : i8
+// CHECK:             %[[VAL_8:.*]] = cmpi eq, %[[VAL_7]], %[[VAL_2]] : i8
+// CHECK:             %[[VAL_9:.*]] = sexti %[[VAL_8]] : i1 to i8
+// CHECK:             %[[VAL_10:.*]] = and %[[VAL_9]], %[[VAL_4]] : i8
+// CHECK:             graphblas.yield transform_out %[[VAL_10]] : i8
 // CHECK:           }
-// CHECK:           return %[[VAL_5:.*]] : tensor<?x?xi8, #sparse_tensor.encoding<{ dimLevelType = [ "dense", "compressed" ], dimOrdering = affine_map<(d0, d1) -> (d0, d1)>, pointerBitWidth = 64, indexBitWidth = 64 }>>
+// CHECK:           return %[[VAL_11:.*]] : tensor<?x?xi8, #sparse_tensor.encoding<{ dimLevelType = [ "dense", "compressed" ], dimOrdering = affine_map<(d0, d1) -> (d0, d1)>, pointerBitWidth = 64, indexBitWidth = 64 }>>
 // CHECK:         }
 
     func @apply_minv_int_matrix(%sparse_tensor: tensor<?x?xi8, #CSR64>) -> tensor<?x?xi8, #CSR64> {
         %answer = graphblas.apply %sparse_tensor { apply_operator = "minv" } : (tensor<?x?xi8, #CSR64>) to tensor<?x?xi8, #CSR64>
         return %answer : tensor<?x?xi8, #CSR64>
     }
 
-// CHECK-LABEL:   func @apply_minv_int_vector(
+// CHECK-LABEL:   builtin.func @apply_minv_int_vector(
```

Comment:
Drop `builtin.`

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/35a3d57cb129d01978886379d0c1aab19388680d implements this suggestion.